packages/remix-node/fetch.ts

```diff
@@ -72,5 +72,7 @@ export const fetch: typeof webFetch = (
     ...init,
   };
 
-  return webFetch(info, init as RequestInit);
+  let webResponse = await webFetch(info, init as RequestInit);
```

Comment:
Doesn't this change the expectation of `fetch`, since now you `await` the response before returning? What if the original response is streaming?